tests/e2e/release/cht-config/contact-edit.specs.js

```diff
@@ -58,4 +59,50 @@ describe('Editing contacts with the CHT config', () => {
     await contactPage.selectLHSRowByText(healtchCenterName);
     expect(await contactPage.peopleRows.count()).toBe(0);
   });
+
+  it('should change primary contact', async () => {
+    const contacts = [
+      {
+        _id: 'three_district',
+        type: 'district_hospital',
+        name: 'Maria\'s district',
```

Comment:
You can do `` `Maria's District` `` using back ticks and avoid the escape character.